apps/namadillo/src/App/Ibc/IbcTransfer.tsx

```diff
@@ -93,9 +93,8 @@ export const IbcTransfer = (): JSX.Element => {
         asset,
         chainRegistry.assets.assets
       );
-
       // Include if asset has a corresponding Namada asset, and it's either native or native for namada
-      if (namadaAsset && (!asset.traces || !namadaAsset.traces)) {
```

Comment:
Loosened restrictions here but I don't think it's an issue. 

Can see here from my Keplr I've got USDC on Cosmos Hub and OSMO on Stride and neither are showing up:

<img width="344" height="143" alt="Screenshot 2025-07-12 at 10 22 17 PM" src="https://github.com/user-attachments/assets/f939ffb9-ba2d-42ad-b9ba-facd777d637b" />


<img width="547" height="352" alt="Screenshot 2025-07-12 at 10 22 29 PM" src="https://github.com/user-attachments/assets/ec42c3db-8468-4be9-b909-085e06169ccb" />

<img width="564" height="389" alt="Screenshot 2025-07-12 at 10 24 36 PM" src="https://github.com/user-attachments/assets/47a06ece-5805-4aff-a8f5-d1b17d8e9d42" />